package.json

```diff
@@ -13,7 +13,7 @@
 		"url": "https://github.com/dojo/shim.git"
 	},
 	"scripts": {
-		"prepublish": "grunt dist",
+		"prepublish": "grunt peerDepInstall dist",
```

Comment:
I had originally suggested adding to the `dist` / `dev` grunt pipelines but I remember that there was some push back. I would be happy to add them to the grunt tasks directly.

You are right, I will actually remove this and raise a separate issue and pull request.